clmm/utils.py

```diff
@@ -30,6 +30,10 @@ def compute_radial_averages(xvals, yvals, xbins, error_model='std/sqrt_n'):
         Error on the mean y value in each bin. Specified by error_model
     n : array_like
         Number of objects in each bin
+    binnumber: 1-D ndarray of ints
```

Comment:
Would it be more self-consistent to keep this as xbins being returned?  (there is redundancy between xbins or binnumber, yes?)

Reply:
Unless I misunderstood your point, no, they provide different information. `xbins`gives the bin edges. `binnumber` gives, for each galaxy, the indices of the bin it falls into. This is needed is the user requires to keep track of which galaxy belong to which bin.